From 0a2f1b258032acf84938f2128788d80d90680167 Mon Sep 17 00:00:00 2001
From: Samuel Martin <s.martin49@gmail.com>
Date: Mon, 5 May 2014 14:19:04 +0200
Subject: [PATCH 2/3] dynamicEDT3D: rename CMake module file names

dynamicEDT3D was the only subproject naming its CMake module files
<project name>Config.cmake.
This patch cleans this up, using the <project name>-config.cmake pattern;
so make dynamicEDT3D consistent with what is done for octomap and octovis.

Signed-off-by: Samuel Martin <s.martin49@gmail.com>
---
 dynamicEDT3D/CMakeLists.txt                        | 22 +++++++++++-----------
 ...ake.in => dynamicEDT3D-config-version.cmake.in} |  0
 ...onfig.cmake.in => dynamicEDT3D-config.cmake.in} |  0
 3 files changed, 11 insertions(+), 11 deletions(-)
 rename dynamicEDT3D/{dynamicEDT3DConfig-version.cmake.in => dynamicEDT3D-config-version.cmake.in} (100%)
 rename dynamicEDT3D/{dynamicEDT3DConfig.cmake.in => dynamicEDT3D-config.cmake.in} (100%)

diff --git a/dynamicEDT3D/CMakeLists.txt b/dynamicEDT3D/CMakeLists.txt
index 4e7d96a..f9e77cf 100644
--- a/dynamicEDT3D/CMakeLists.txt
+++ b/dynamicEDT3D/CMakeLists.txt
@@ -67,33 +67,33 @@ install(FILES ${dynamicEDT3D_HDRS}	DESTINATION include/dynamicEDT3D)
 # (this registers the build-tree with a global CMake-registry)
 export(PACKAGE dynamicEDT3D)
  
-# Create a dynamicEDT3DConfig.cmake file for the use from the build tree
+# Create a dynamicEDT3D-config.cmake file for the use from the build tree
 set(DYNAMICEDT3D_INCLUDE_DIRS "${INCLUDE_DIRS}")
 set(DYNAMICEDT3D_LIB_DIR "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 # not used right now (export depends?)
 #set(DYNEDT3D_CMAKE_DIR "${PROJECT_BINARY_DIR}")
-configure_file(dynamicEDT3DConfig.cmake.in
-  "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/cmake/dynamicEDT3D/dynamicEDT3DConfig.cmake" @ONLY)
-configure_file(dynamicEDT3DConfig-version.cmake.in
-  "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/cmake/dynamicEDT3D/dynamicEDT3DConfig-version.cmake" @ONLY)
+configure_file(dynamicEDT3D-config.cmake.in
+  "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/cmake/dynamicEDT3D/dynamicEDT3D-config.cmake" @ONLY)
+configure_file(dynamicEDT3D-config-version.cmake.in
+  "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/cmake/dynamicEDT3D/dynamicEDT3D-config-version.cmake" @ONLY)
 
 # Install the export set for use with the install-tree
 #install(EXPORT FooBarLibraryDepends DESTINATION
 #  "${INSTALL_DATA_DIR}/FooBar/CMake"
 #  COMPONENT dev)
  
-# Create a dynamicEDT3DConfig.cmake file for the use from the install tree
+# Create a dynamicEDT3D-config.cmake file for the use from the install tree
 # and install it
 set(DYNAMICEDT3D_INCLUDE_DIRS "${CMAKE_INSTALL_PREFIX}/include")
 set(DYNAMICEDT3D_LIB_DIR "${CMAKE_INSTALL_PREFIX}/lib")
 #set(DYNAMICEDT3D_CMAKE_DIR "${INSTALL_DATA_DIR}/FooBar/CMake")
-configure_file(dynamicEDT3DConfig.cmake.in
-  "${PROJECT_BINARY_DIR}/InstallFiles/dynamicEDT3DConfig.cmake" @ONLY)
-configure_file(dynamicEDT3DConfig-version.cmake.in
-  "${PROJECT_BINARY_DIR}/InstallFiles/dynamicEDT3DConfig-version.cmake" @ONLY)
+configure_file(dynamicEDT3D-config.cmake.in
+  "${PROJECT_BINARY_DIR}/InstallFiles/dynamicEDT3D-config.cmake" @ONLY)
+configure_file(dynamicEDT3D-config-version.cmake.in
+  "${PROJECT_BINARY_DIR}/InstallFiles/dynamicEDT3D-config-version.cmake" @ONLY)
 install(FILES
   "${PROJECT_BINARY_DIR}/InstallFiles/dynamicEDT3DConfig.cmake"
-  "${PROJECT_BINARY_DIR}/InstallFiles/dynamicEDT3DConfig-version.cmake" 
+  "${PROJECT_BINARY_DIR}/InstallFiles/dynamicEDT3DConfig-version.cmake"
   DESTINATION share/dynamicEDT3D/)
 
 # Write pkgconfig-file:
diff --git a/dynamicEDT3D/dynamicEDT3DConfig-version.cmake.in b/dynamicEDT3D/dynamicEDT3D-config-version.cmake.in
similarity index 100%
rename from dynamicEDT3D/dynamicEDT3DConfig-version.cmake.in
rename to dynamicEDT3D/dynamicEDT3D-config-version.cmake.in
diff --git a/dynamicEDT3D/dynamicEDT3DConfig.cmake.in b/dynamicEDT3D/dynamicEDT3D-config.cmake.in
similarity index 100%
rename from dynamicEDT3D/dynamicEDT3DConfig.cmake.in
rename to dynamicEDT3D/dynamicEDT3D-config.cmake.in
-- 
1.9.2

